src/cli/commands/test/formatters/format-test-results.ts

```diff
@@ -63,9 +64,13 @@ export function extractDataToSendFromResults(
   let sarifData = {};
   let stringifiedSarifData = '';
   if (options.sarif || options['sarif-file-output']) {
-    sarifData = !options.iac
-      ? createSarifOutputForContainers(results)
-      : createSarifOutputForIac(results);
+    if (options.iac) {
+      sarifData = createSarifOutputForIac(results);
+    } else if (!options.docker) {
```

Comment:
Can you make this 
```suggestion
    } else if (options.docker) {
```
I feel like the default should be createSarifOutputForOpenSource, and not for Containers.